lib/src/components/Fields/Section.tsx

```diff
@@ -12,19 +15,32 @@ const messages = defineMessages({
   },
 });
 
+/**
+ * Manage conditions to display or not the dropzone at the top of a section
+ * It must not display a dropzone in a section that contain other sections to
+ * avoid using widgets and sections at the same level
+ *
+ * @param id of the section
+ * @param formSchema is the whole schema loaded in the library
+ * @param selector is the section name if one is selected
+ */
+function checkObjectDeepness(id: string, formSchema: VernaSchemaType, selector: Maybe<string>) {
+  let containObject = false;
+  _.forEach(formSchema.schema?.properties, (section) => {
+    if (section.type === 'object') containObject = true;
+  });
+  return containObject && !selector && id == 'root';
+}
+
 export default function Section({
   title,
   idSchema,
   properties,
   description,
 }: ObjectFieldTemplateProps) {
-  const { isEditor, selector, DropZone } = useVerna();
+  const { isEditor, DropZone, formSchema, selector } = useVerna();
   const [isEditingParameters, setIsEditingParameters] = useState(false);
-  const isRoot = idSchema.$id === 'root';
-
-  // Display drop zone at the top of the section only if in editor mode and if it's not the
-  // primary section in the case where there aren't any selector
-  const ownDropZone = isEditor && !(!selector && isRoot);
+  const hasSection = checkObjectDeepness(idSchema['$id'], formSchema, selector);
```

Comment:
`checkObjectDeepness` seems not to be the right name to this method. Maybe `doesContainSection` ?